client/my-sites/marketing/tools/header.tsx

```diff
@@ -40,7 +39,7 @@ const MarketingToolsHeader: FunctionComponent< Props > = ( { handleButtonClick }
 
 				<div className="tools__header-button-row">
 					<Button onClick={ handleButtonClick } primary>
-						{ translate( 'Boost My Traffic' ) }
+						{ translate( 'Boost my traffic' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.